drbd_metrics_test.go

```diff
@@ -166,22 +166,46 @@ func TestDrbdParsing(t *testing.T) {
 		t.Errorf("lower-pending should be 2")
 	}
 
+	if true != drbdDevs[0].Devices[0].Quorum {
+		t.Errorf("quorum should be true")
+	}
+
+	if true != drbdDevs[1].Devices[0].Quorum {
+		t.Errorf("quorum should be true")
+	}
+
 	if 456 != drbdDevs[0].Connections[0].PeerDevices[0].Received {
 		t.Errorf("received should be 456")
 	}
 
+	if 456 != drbdDevs[1].Connections[0].PeerDevices[0].Received {
```

Comment:
Can you separate the UT from the previous PR in separated PR, or at least a new commit? It helps to better organize the changes and the repo history.

Reply:
Sure thing! Thanks for the quick feedback.

Reply:
Removed the tests in a subsequent commit. I'll issue another PR to fix these up later on. Thanks again!

Reply:
thx